### PR TITLE
Lecture deps, removing unused packages and dependencies for the lisp lecture.

### PR DESCRIPTION
**Pull request by artnie**

### CODE REVIEW

**Comment by artnie** [comment]
The dependency to json_prolog_msgs-srv will be deleted asap.

**Comment by gaya-** [comment]
@artnie doch am arbeiten. I'm not sure if that can be done so easily. Besides, json_prolog_msgs are in the iai_common_msgs repository, which is not so big. So, zur Not, leave it there and add iai_common_msgs. See for yourself.

File: cram-14.04.rosinstall [hunk 1]
```diff
@@ -3,40 +3,16 @@
...
-    local-name: iai_maps
-    uri: https://github.com/code-iai/iai_maps.git
-    version: 6a4f3fab1fa836fdf26e7ea9a36ab140c2e2d563
-
-############## IAI common msgs repository ############
+    
+############## IAI services and messages ###########
 - git:
-    local-name: iai_common_msgs
+    local-name: iai-messages
```
Comment (gaya-) [inline cram-14.04.rosinstall]: Please rename this back to iai_common_msgs and do use the version, as the messages can change and break compilation.

File: cram-16.04.rosinstall [hunk 2]
```diff
@@ -3,38 +3,16 @@
...
-############## PR2 inverse kinematics solver #########
-- git:
-    local-name: pr2_kinematics
-    uri: https://github.com/code-iai/pr2_kinematics.git
-    version: kinetic-devel
-# - git:
-#     local-name: moveit_msgs
-#     uri: https://github.com/ros-planning/moveit_msgs.git
-#     version: kinetic-devel
+    version: master
```
Comment (gaya-) [inline cram-16.04.rosinstall]: Same here

File: cram_knowrob/cram_semantic_map_utils/package.xml [hunk 3]
```diff
@@ -19,13 +19,13 @@
...
   <build_depend>roslisp</build_depend>
-  <build_depend>cram_json_prolog</build_depend>
+  <!--build_depend>cram_json_prolog</build_depend-->
   <build_depend>cram_utilities</build_depend>
   <build_depend>cl_transforms</build_depend>
 
   <run_depend>roslisp_utilities</run_depend>
   <run_depend>roslisp</run_depend>
-  <run_depend>cram_json_prolog</run_depend>
+  <!--run_depend>cram_json_prolog</run_depend-->
```
Comment (gaya-) [inline cram_knowrob/cram_semantic_map_utils/package.xml]: Getting rid of package.xml dependency on "cram_json_prolog" breaks loading of asdf system "cram-json-prolog". Besides, if the "cram_json_prolog" is just catkin_ignored, "(asdf:load-system :cram-json-prolog)" still works, so it's still being loaded. Please get rid of the catkin_ignore in cram_json_prolog and bring the dependencies on "cram_json_prolog" back in these package.xml-s. As long as the "iai_common_msgs" is there, "cram_json_prolog" doesn't create any problems.

**Pull request merged**